pages/about.html

```diff
@@ -0,0 +1,154 @@
+      <div class="heroContent contactEmail">
+        <p>Please contact us on our various platforms for any further questions about our programs for 2023! </p>
+        <a href="mailto:ginoschool@gmail.com? subject=Enquiry for Admission">ginoschool@gmail.com</a>
+      </div>
+      <div id="about"></div>
+    </section>
+
+    <!-- ========= About School Section ===================== -->
+
+    <section class="schoolInfo">
+      <div class="schoolDetails">
+        <h1>About our <br> Videography School</h1>
+        <hr>
+        <p>Our videography school stands at the forefront of innovation, empowering aspiring videographers to capture the world through a unique blend of technical mastery and artistic expression.</p>
+      </div>
+      <div class="footer aboutSchool">
+        <div class="footerLogo"><a href="#"><img src="../assets/logo1.png" alt=""></a>
+          <p id="about-description">The Excellence of <br>videography</p>
+        </div>
+        <div id="graduates"></div>
+      </div>
+    </section>
+
+    <!-- =================== Graduates Section =================== -->
+
+    <section class="schoolInfo">
+      <div class="schoolDetails">
+        <h1>Celebrating our Accomplished Graduates</h1>
+        <hr>
+        <p>Our videography school produces remarkable graduates who excel in capturing unforgettable moments and telling compelling visual stories, leaving a lasting impact in the industry.</p>
+      </div>
+
+      <div class="graduatesContainer">
+        <div class="graduatesCard card1">
+          <div class="graduatesContent">
+            <h1>2022</h1>
+            <p>Celebrating the journey of our talented graduates.</p>
+          </div>
+        </div>
+        <div class="graduatesCard card2">
+          <div class="graduatesContent">
+            <h1>2021</h1>
+            <p>Exploring new horizons with our successful alumni.</p>
+          </div>
+        </div>
+      </div>
+      
+      
+    </section>
+
+ <!-- ========== Sponsor ================== -->
+
+    <section class="sponsorContainer" id="sponsor">
+      <div class="sponsorHeading">
+        <h1>Sponsor</h1>
+        <hr>
+      </div>  
+    <div class="sponsorGrid">
+      <div class="sponsorCard">
+        <i class="fa-solid fa-blog fa-xl"></i><p>Blog</p>
+      </div>
+      <div class="sponsorCard">
+        <i class="fa-brands fa-google fa-xl"></i><p>Google</p>
+      </div>
+      <div class="sponsorCard">
+        <i class="fa-brands fa-joomla fa-xl"></i><p>Joomla</p>
+      </div>
+      <div class="sponsorCard">
+        <i class="fa-brands fa-stumbleupon fa-xl"></i><p>Stumbleupon</p>
+      </div>
+      <div class="sponsorCard">
+        <i class="fa-brands fa-yahoo fa-xl"></i><p>Yahoo</p>
+      </div>
+    </div>
+    
+    </section>
+
+  </main>
+
+  <footer>
+    <div class="footerLogo"><a href="#"><img src="../assets/logo1.png" alt=""></a>
+      <p>2023 Gino Videography <br>All rights reserved</p>
+    </div>
+  </footer>
```

Comment:
- [ Required ] I think it would be better if the `footer` section of your `about.html` page has a dark background color similar to the way it is in the Behance design. Currently, the background color is white which is not consistent with the design on Behance, please see the screenshot below and rectify this.

## Behance design
![Screenshot from 2023-06-01 04-31-51](https://github.com/Izebiafe/videography-capstone/assets/16392046/90bdc5f4-86b6-43c2-ab74-9d0f8cc79b7a)

## Your design
![Screenshot from 2023-06-01 04-27-59](https://github.com/Izebiafe/videography-capstone/assets/16392046/ba4241be-9752-4a16-83b9-58dcb4844a34)